### ISSUE TITLE
Built-in @aws-sdk/client-s3 Body transform* functions transform self rather than body

### ISSUE THREAD

**Issue by kazdegroot** [awslabs/llrt#191]
This was working in either 0.1.6 or 0.1.7, but with 0.1.8 this code

```js
        const command = new GetObjectCommand({
                Bucket: BUCKET,
                Key: "index.html",
            })
        const response = await s3.send(command);
        console.log({
            response,
            bodyString: await response.Body.transformToString(),
            bodyArray: await response.Body.transformToByteArray()
        })
```
logs
```js
{
  response: {
    $metadata: { httpStatusCode: 200, ..., cfId: undefined, attempts: 1, totalRetryDelay: 0 },
    AcceptRanges: 'bytes',
    LastModified: 2024-02-21T15:09:47.000Z,
    ContentLength: 582,
    ...
    CacheControl: 'no-cache,max-age=0,public',
    ContentType: 'text/html',
    ServerSideEncryption: 'AES256',
    Metadata: {},
    Body: { transformToByteArray: [function: Me], transformToString: [function: Re], transformToWebStream: [function: Ie] }
  },
  bodyString: '[object Object]',
  bodyArray: [Circular]
}
```

Seems like the transform* functions are operating on the Body object with functions, rather than on the responses actual Body.

**Comment by richarddavison** [comment]
Thank you so much for the report. Fixed in 0.1.9-beta, releasing now: https://github.com/awslabs/llrt/actions/runs/8011479816

**Comment by kazdegroot** [comment]
Thank you for llrt and fixing this so very quickly!
